test-suite/src/test/scala/org/scalajs/testsuite/javalib/CollectionTest.scala

```diff
@@ -93,7 +91,11 @@ trait CollectionTest[F <: CollectionFactory] extends JasmineTest {
     }
 
     it("should store custom objects") {
-      case class TestObj(num: Int)
+      case class TestObj(num: Int) extends jl.Comparable[TestObj] {
+        def compareTo(o: TestObj): Int =
+          if (o==null) -1
```

Comment:
For collections that can use `Comparator` (by default the natural ordering comparator) to locate an element. `SortedSet`s fall in this category.